sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalogSuite.scala

```diff
@@ -1440,6 +1441,8 @@ abstract class SessionCatalogSuite extends AnalysisTest {
         }
 
         assert(cause.getMessage.contains("Undefined function: 'undefined_fn'"))
+        // SPARK-21318: the error message should contains the current database name
```

Comment:
what's the full error message?

Reply:
org.apache.spark.sql.AnalysisException: Undefined function: 'undefined_fn'. This function is neither a registered temporary function nor a permanent function registered in the database 'db1'.;